.circleci/config.yml

```diff
@@ -1233,7 +1193,7 @@ jobs:
       - update-spm-installation-commit
       - run:
           name: Submit Purchase Tester
-          command: bundle exec fastlane deploy_purchase_tester dry_run:true
+          command: bundle exec fastlane deploy_purchase_tester dry_run:<< parameters.dry_run >>
```

Comment:
and this is the actual meat of the PR: being able to define a parameter at the job level easily and then use it